.pre-commit-config.yaml

```diff
@@ -41,3 +41,39 @@ repos:
         # needed to make excludes in pyproject.toml work
         # see here https://github.com/econchick/interrogate/issues/60#issuecomment-735436566
         pass_filenames: false
+  - repo: local
+    hooks:
+      - id: convert-notebooks
+        name: Convert Notebooks to Markdown
+        entry: python ./docs/source/.codespell/notebook_to_markdown.py
+        language: python
+        pass_filenames: false
+        always_run: false
+        additional_dependencies: ["nbconvert", "nbformat"]
+        args: ["--tempdir", "tmp_markdown"]
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.3.0
+    hooks:
+      - id: codespell
+        args: [
```

Comment:
Noticing that codespell can be configured by pyproject.toml. I think it's worth standardizing on `pyproject.toml` as the place for configuration. Establishing the pattern will be good for the long-term health of the package. @drbenvincent what are your thoughts here?

Reply:
Agree - `pyproject.toml` should do as much of the project config work as possible.

Reply:
yea that's an easy fix